lib/android/app/src/main/java/com/spruceid/didkitexample/MainActivity.kt

```diff
@@ -0,0 +1,27 @@
+package com.spruceid.didkitexample
```

Comment:
Is it normal for an example to be under `/src`?

Reply:
The `/src` is how the Android Studio project is structured, the exemple is actually under the `/app` module.

As there is an example folder in the repo, maybe it makes more sense to move the android app to the example folder, what do you think?

Reply:
heh tried to go too fast. Maybe `/example` would be more explicit but I suppose `/app` is the standard

Reply:
Yeah, `app` is the standard, but `example` is a better name.

Reply:
@sbihel renamed to `example` 😁